doc/QUICKLOOK/snr_qa.rst

```diff
@@ -0,0 +1,59 @@
+====================
+qa-CAMERA-EXPID.yaml
+====================
+
+:Summary: This QA for QuickLook first includes the calculation of 
+        signal-to-noise in the spectrum after sky subtraction. 
+:Naming Convention: ``qa-{ARM}{SPECTROGRAPH}-{EXPID}.yaml``, where 
+        {ARM} is the 1-char arm name (r,b,z), {SPECTROGRAPH} indexes 
+        CCDs 0-9 on that arm, and {EXPID} is the 8-digit exposure ID.  
+        Together, {ARM}{SPECTROGRAPH} specify a {CAMERA}.
+:Regex: ``qa-[brz][0-9]-[0-9]{8}\.yaml``
+:File Type:  YAML
+
+
+Inputs
+======
+
+Written by qa_quicklook.py, with Calculate_SNR using:
+
+- frame
+- skymodel
+
+Contents
+========
+
+========== ================ ===========================
+KEYNAME    Type             Contents
+========== ================ ===========================
+SNR        Py dictionary    Spectral signal-to-noise
+========== ================ ===========================
+
+
+
+Dictionary Contents
+===================
+
+KEYNAME = SNR
+
+Spectral signal-to-noise calculations per fiber and per CCD amp.
+
+Required Header Keywords
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+================ ============= ========== ============
+KEY              Example Value Type       Comment
+================ ============= ========== ============
+ARM              r             char       Spectrograph arm b,r,z

```

Comment:
Hello Ben,

In looking at the .rst file, and the web-output, it is not clear what 
spacing you are referring to?  Can you please clarify?

Bob

--------------------------------------------------------------------------
Robert Kehoe					      (214)768-1793 (off.)
Professor of Physics
Department of Physics				       (214)768-4095 (fax)
Southern Methodist University			     kehoe@physics.smu.edu
Dallas, TX 75275		          http://www.physics.smu.edu/kehoe

On Wed, 1 Jun 2016, Benjamin Alan Weaver wrote:

> 
> In doc/QUICKLOOK/snr_qa.rst:
> 
> > +
> > +
> > +Dictionary Contents
> > +===================
> > +
> > +KEYNAME = SNR
> > +
> > +Spectral signal-to-noise calculations per fiber and per CCD amp.
> > +
> > +Required Header Keywords
> > +~~~~~~~~~~~~~~~~~~~~~~~~
> > +
> > +================ ============= ========== ============
> > +KEY              Example Value Type       Comment
> > +================ ============= ========== ============
> > +ARM              r             char       Spectrograph arm b,r,z
> 
> The spacing in this table looks a little off.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the
> thread.[ASng8cPlCNvib2HMI7QPpSLXQdUCiD-lks5qHcLFgaJpZM4IrvK6.gif]
> 
> 
>